gatsby-ssr.js

```diff
@@ -2,21 +2,33 @@ import React from 'react'
 import { WrapPagesWithLocaleContext } from './src/components/localization'
 import './src/components/localization/config'
 import { MediaContextProvider } from './src/themes/media'
-import { DerivProvider } from './src/store'
+import {
+    AcademyProvider,
+    BreakpointsProvider,
+    PopupProvider,
+    RegionProvider,
+    WebsiteStatusProvider,
+} from './src/store'
 
 export const wrapRootElement = ({ element }) => {
     return (
-        <DerivProvider>
-            <MediaContextProvider>{element}</MediaContextProvider>
-        </DerivProvider>
+        <AcademyProvider>
```

Comment:
@mohammad-rework You can use the same `Providers` components here instead of duplicating the same code.